Challenges401/Arrays/Search.cs

```diff
@@ -1,10 +1,31 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace Challenges401.Arrays
+﻿namespace Challenges401.Arrays
 {
     class Search
     {
+        public static int BiSearch(int value, int[] array)
```

Comment:
The order of parameters here _BiSearch(int value, int[] array)_ doesn't match up with how you call the method in your test _BiSearch(testArray, testN)_.  Switch this around to be _BiSearch(testN, testArray)_ and you'll be good to pass the build.